decompressed/gui_file/www/docroot/modals/diagnostics-connection-modal.lp

```diff
@@ -354,10 +354,11 @@ else
 
     if xdsl_params["dsl_status"] == "Up" then
         statusIconDSL = "<i class=\"icon-ok icon-large green\"></i> "
+        html[#html+1] = ui_helper.createLabel(T"DSL Status",format("%s %s",statusIconDSL, T"Up", attributes)
     else
         statusIconDSL = "<i class=\"icon-ban-circle icon-large red\"></i> "
+        html[#html+1] = ui_helper.createLabel(T"DSL Status",format("%s %s",statusIconDSL, T"Not up", attributes)
     end
-    html[#html+1] = ui_helper.createLabel(T"DSL Status",format("%s %s",statusIconDSL, xdsl_params["dsl_status"]), attributes)
```

Comment:
meglio cambiare qui semplicemente in `T(xdsl_params["dsl_status"])` piuttosto che copiare il codice due volte

Reply:
@FrancYescO 

Come preferisci e secondo te va meglio.